.github/workflows/go.yml

```diff
@@ -0,0 +1,28 @@
+name: Go
+on: [pull_request]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.14
```

Comment:
```suggestion
    - name: Set up Go 1.15
```

Reply:
I wasn't sure about that one, but that's great